packages/fx-core/templates/plugins/resource/bot/bicep/botProvision.template.bicep

```diff
@@ -8,7 +8,7 @@ var botServiceName = contains(provisionParameters, 'botServiceName') ? provision
 var botServiceSku = contains(provisionParameters, 'botServiceSku') ? provisionParameters['botServiceSku'] : 'F0' // Try to read SKU for Azure Bot Service from parameters
 var botDisplayName = contains(provisionParameters, 'botDisplayName') ? provisionParameters['botDisplayName'] : '${resourceBaseName}' // Try to read display name for Azure Bot Service from parameters
 var serverfarmsName = contains(provisionParameters, 'botServerfarmsName') ? provisionParameters['botServerfarmsName'] : '${resourceBaseName}bot' // Try to read name for App Service Plan from parameters
-var webAppSKU = contains(provisionParameters, 'botWebAppSKU') ? provisionParameters['botWebAppSKU'] : 'F1' // Try to read SKU for Azure Web App from parameters
+var webAppSKU = contains(provisionParameters, 'botWebAppSKU') ? provisionParameters['botWebAppSKU'] : 'B1' // Try to read SKU for Azure Web App from parameters
```

Comment:
Why changing F1 to B1? Is there any background about this change?

Reply:
This is to enable `Always On`.
- If `Always On` is disabled, the app automatically stops after 20 minutes idle. For our bot scenario it may cause cold start / timeout issue.
- `Always On` requires basic or higher app service plan, so here we choose B1.

Reply:
One last reminder - maybe we need to inform customers about this change since it's kind of related to billing.

Reply:
Good point. @XiaofuHuang - is it possible to conditionally use `B1` only when the preview feature flag is enabled?
For bicep file we do support `if` statement such as `{{#if (contains "fx-resource-bot" plugins) }}`.

Reply:
If we want to enable `alwaysOn`, the SKU should be changed to `B1`

Reply:
I mean only enable alwaysOn when the preview feature flag is enabled.